frontend/cypress/integration/multiPart.spec.js

```diff
@@ -0,0 +1,296 @@
+import faker from 'faker'
```

Comment:
Why is this file called `multiPart`? Do you mean `multiProject`?

Reply:
Resolved in 2fd171f.